### ISSUE TITLE
Combining Push Notifications with In-App Notifications

### ISSUE THREAD

**Issue by moustafadevrn** [infobip/mobile-messaging-react-native-plugin#73]
**Description:**
When combining push notifications with in-app notifications, the in-app notification appears too early when the push notification is received and clicked while the app is closed. The in-app notification appears parallel to the splash screen, which may not provide the optimal user experience.

**Steps to Reproduce:**
Combine push notifications with in-app notifications.
Send a notification while the app is closed.
Click on the push notification to open the app.

**Expected Behavior:**
The in-app notification should appear after the home screen fully loads, rather than during the splash screen.

**Actual Behavior:**
The in-app notification appears in parallel with the splash screen.

**Comment by fortesdev** [comment]
Hi @moustafadevrn, I am afraid the control for displaying manually the notification in-app notification is only exposed on native Android and iOS. On React Native it is displayed "automatically", as soon as it can.

**Comment by riskpp** [comment]
Hi @moustafadevrn, as a workaround you may consider building a [Flow](https://www.infobip.com/docs/moments/manage-flow#create) for an in-app notification campaign: 
- listen to app launch event
- put pause element after this
- send in-app notification

**Comment by moustafadevrn** [comment]
Hi @riskpp , I followed the recommendation and attempted to create a flow on the dashboard to trigger the in-app notification after a delay of 10 seconds upon app launch. However, I encountered difficulties as the in-app notification was not triggered as expected. Despite configuring the flow to listen to the app launch event and inserting a pause element, the in-app notification did not appear after the specified delay.

Additionally, it would greatly streamline our processes if we could utilize the existing option to manage both types of notifications in a single interface. Specifically, leveraging the checkbox to include in-app notifications when creating broadcast notifications would consolidate our workflow and enhance efficiency, eliminating the need to manage notifications in two separate locations.
I appreciate your assistance and would be grateful for any further guidance or solutions you can provide to address this issue.

**Comment by riskpp** [comment]
Hello @moustafadevrn , could you please contact your account manager regarding this issue, might be easier to resolve.

**Comment by ikresicc** [comment]
Closing this issue. If needed, new task can be opened.
